src/eval_tool.py

```diff
@@ -0,0 +1,64 @@
+import cPickle
```

Comment:
Include the future copypasta?

```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

Reply:
Sure. Is it just to be safe?

Reply:
Yeah, in particular the division thing. I often forgot to copy paste these imports, and things work. Later I expand the script/notebook, and happen to use division, expecting it work like the __future__, and get weird bugs.